Resources/Prototypes/Nyanotrasen/Guidebook/epistemics.yml

```diff
@@ -5,7 +5,7 @@
 
 - type: guideEntry
   id: AltarsGolemancy
-  name: guide-entry-altars-golemancy
+  name: Altars
```

Comment:
Change back to `guide-entry-altars-golemancy` and edit the localized string. Should be in the file `Resources/Locale/en-US/nyanotrasen/guidebook/guides.ftl`. Its in our namespace so don't worry about comments.

Can also get rid of the `golemancy` in both places. Not really a big deal though.